spec/models/planned_disbursement_spec.rb

```diff
@@ -9,6 +9,40 @@
     it { should validate_presence_of(:currency) }
     it { should validate_presence_of(:value) }
 
+    it "does not allow two original planned disbursements for the same activity, financial quarter and year" do
+      beis = create(:beis_organisation)
+      report = create(:report)
+      first_planned_disbursement = create(:planned_disbursement, parent_activity: activity, financial_quarter: 1, financial_year: 2020)
+      second_planned_disbursement = PlannedDisbursement.new(
+        value: 1000,
+        planned_disbursement_type: :original,
+        financial_quarter: 1,
+        financial_year: 2020,
+        period_start_date: Date.parse("2020-04-01"),
+        providing_organisation_name: beis.name,
+        providing_organisation_type: beis.organisation_type,
+        providing_organisation_reference: beis.iati_reference,
+        currency: "GBP",
+        parent_activity: activity,
+        report: report
+      )
+
+      error_message = t("activerecord.errors.models.planned_disbursement.attributes.planned_disbursement_type.only_one_original",
+        financial_quarter: first_planned_disbursement.financial_quarter,
+        financial_year_start: first_planned_disbursement.financial_year,
+        financial_year_end: first_planned_disbursement.financial_year + 1)
+
+      expect(second_planned_disbursement).to be_invalid
+      expect(second_planned_disbursement.errors[:base]).to eq [error_message]
```

Comment:
Never seen an expectation with `eq [something_between_square_brackets]` before. Is this because `error_message` is a variable that has several items in it? (I mean, the translation file of the error needs a financial_quarter, financial_year start and end)

Reply:
No you haven't you are right! last_financial_quarter should be 1 2020. Thank you! I hope this is all a bit clearer when we can remove the dates in the next work.

Reply:
I think it is because `errors[:base]` is an array of error messages, but we have setup the spec so that we can expect only one, you can probably wrtie this expecation a number of ways, perhaps `include?(error_message)` would be clearer?

Because I wanted to show the financial quarter and year that already has an original, I had to jump through the hoops of creating the expect error string so I can compare in the final expectation - I can see how this might make you think there is more going on - not sure what to do about that, I was keen to test that the message was correct.

Here is the guide tor the errors part:
https://guides.rubyonrails.org/active_record_validations.html#errors

Reply:
No, that's absolutely fine. I wasn't expecting you to change the way you wrote this expectation. It was just curiosity. Glad to see that my guess (square brackets because it's an array) was right. Thanks for the supporting documentation and explanation though!